### ISSUE TITLE
Emphasis management error

### ISSUE THREAD

**Issue by jdecool** [erusev/parsedown#198]
In a document, I have this markdown :

```
this_text_is_normal 
_this text_is italic_
```

I expect something like :

```
<p>this_text_is_normal 
<em>this text_is italic</em></p>
```

But parsedown generate :

```
<p>this_text_is_normal
_this text<em>is italic</em></p>
```

**Comment by meitar** [comment]
I just ran into a similar issue to this where words have control characters in them. For example, the decentralized social network Diaspora* is formally written with an asterisk in the name (i.e., the correct typographical representation of the project is `Diaspora*` with an asterisk, not `Diaspora` with no asterisk). The same is true for some renderings of words in sociology (like `trans*`) and, of course, other areas of more technical specializations, such as those discussed above.

If I have two paragraphs of text that includes such a phrase, like this:

> Let's query the Diaspora* pod to see what happens.
> 
> Note that Diaspora* has no formal API to speak of, but this query does work, so *shrug*.

My expectation is that only the final word in the second paragraph (`*shrug*`) will be italicized. However, using Parsedown, what actually gets italicized is the text `* has no formal API to speak of, but this query does work, so *`.

Basically this is just…well, incorrect, it seems to me.

**Comment by domsson** [comment]
This seems to be related to issue #703

**Comment by domsson** [comment]
This seems to be related to #703 and #364
